### DISCUSSION TITLE
Stolen repository

### DISCUSSION THREAD

**Discussion by MaksymAndreiev** [community/community#23200] [New to GitHub]
<p>I am a newbie, so I may not understand what is going on. I was browsing repository code and nothing else. Then, my friend asked me, what a strange repository I have in my account. I discovered that that repository is now mine. How? I don’t understand. There is no “forked from” caption. Author is displayed as a contributor. And original repository have been disappeared. So have I stolen this repository? How can I return it to author?</p>

**Comment by canuckjacq** [comment, marked as answer]
<p>:wave: Welcome!</p>
<p>I’ve had a look, and I think I found the repository you’re talking about, and no, don’t worry - it’s much simpler than that.</p>
<p>It looks as though you forked it (this does happen by accident sometimes if you use a keyboard shortcut), so that’s how it got onto your account.</p>
<p>If the owner of a repository makes it unavailable to you - either by making it private or deleting it, etc, then your repository will no longer say “forked from”. It will essentially become its own standalone repository.</p>
<p>If you didn’t mean to fork this repository and you don’t need it, you can simply <a href="https://docs.github.com/en/github/administering-a-repository/deleting-a-repository">delete the repository</a>.</p>
